fe2-android/app/src/mock/java/com/github/dedis/student20_pop/Injection.java

```diff
@@ -0,0 +1,87 @@
+package com.github.dedis.student20_pop;
```

Comment:
what is this file used for?

Reply:
It *will* be used to provide mock instances for unit tests.